project/lit/views.py

```diff
@@ -38,6 +39,49 @@ def get_context_data(self, **kwargs):
         context['tags'] = models.ReferenceFilterTag.get_all_tags(self.assessment.id)
         return context
 
+class BulkTagging(TemplateView):
```

Comment:
I added the TeamMemberOrHigherMixin and verified that it properly 403's if you try to access the URL for an assessment you're not a member of. I wasn't sure if BaseList was appropriate/needed as well? This will be in next commit.